pkg/resource/model/testdata/v1alpha1/readone/observed/late_initialize.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Model
+metadata:
+  creationTimestamp: null
+  name: xgboost-model
+spec:
+  containers:
+  - containerHostname: xgboost
+    environment:
+      my_var: my_value
+      my_var2: my_value2
+    image: 246618743249.dkr.ecr.us-west-2.amazonaws.com/sagemaker-xgboost:1.0-1-cpu-py3
+    imageConfig:
+      repositoryAccessMode: Platform
+      repositoryAuthConfig: {}
```

Comment:
remove?

Reply:
These same four lines are present in the created.yaml of this resource.